docs/source/classification/dice.rst

```diff
@@ -0,0 +1,33 @@
+.. customcarditem::
+   :header: Dice
+   :image: https://pl-flash-data.s3.amazonaws.com/assets/thumbnails/tabular_classification.svg
+   :tags: Classification
+
+.. include:: ../links.rst
+
+####
+Dice
+####
+
+Module Interface
+________________
+
+.. autoclass:: torchmetrics.Dice
+    :noindex:
+
+Functional Interface
+____________________
+
+.. autofunction:: torchmetrics.functional.dice
+    :noindex:
+
+
+##########
+Dice Score
+##########
+
+Functional Interface (will be deprecated)
```

Comment:
@Borda, @SkafteNicki, @justusschock I could not find an error in testing documentation, it turned out that the problem is that Sphinx won't let you use `Functional Interface` twice, so I added `(will be deprecated)`.

But maybe there are better ways to rename `Functional Interface` for `Dice Score`?